lib/pages/chat/events/message.dart

```diff
@@ -211,6 +212,13 @@ class Message extends StatelessWidget {
                                             : FutureBuilder<User?>(
                                                 future: event.fetchSenderUser(),
                                                 builder: (context, snapshot) {
+                                                  // Doesn't show the sender name if the event is encrypted.
+                                                  if (event.type ==
+                                                      EventTypes.Encrypted) {
+                                                    return const SizedBox
+                                                        .shrink();
+                                                  }
```

Comment:
please wait #719 resolve and applied the same

Reply:
done